diff --git forkSrcPrefix/kernel/model/cloud_service.go forkDstPrefix/kernel/model/cloud_service.go
index 8b502e03e..5dc797ee7 100644
--- forkSrcPrefix/kernel/model/cloud_service.go
+++ forkDstPrefix/kernel/model/cloud_service.go
@@ -541,7 +541,7 @@ var (
 	errRequestUserFailed = errors.New("request user failed")
 )
 
-func getUser(token string) (*conf.User, error) {
+func getUserOrigin(token string) (*conf.User, error) {
 	result := map[string]any{}
 	request := httpclient.NewCloudRequest30s()
 	resp, err := request.
@@ -574,6 +574,35 @@ func getUser(token string) (*conf.User, error) {
 	return user, nil
 }
 
+func getUser(token string) (*conf.User, error) {
+	user := &conf.User{
+		UserId:                          "0",
+		UserName:                        "_",
+		UserAvatarURL:                   "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7",
+		UserHomeBImgURL:                 "",
+		UserTitles:                      []*conf.UserTitle{},
+		UserIntro:                       "",
+		UserNickname:                    "",
+		UserCreateTime:                  "29991231 00:00:00",
+		UserSiYuanProExpireTime:         -1,
+		UserToken:                       "token",
+		UserTokenExpireTime:             "32503593600",
+		UserSiYuanRepoSize:              0,
+		UserSiYuanPointExchangeRepoSize: 0,
+		UserSiYuanAssetSize:             0,
+		UserTrafficUpload:              0,
+		UserTrafficDownload:           0,
+		UserTrafficAPIGet:              0,
+		UserTrafficAPIPut:              0,
+		UserTrafficTime:                0,
+		UserSiYuanSubscriptionPlan:     0,
+		UserSiYuanSubscriptionStatus:   0,
+		UserSiYuanSubscriptionType:     1,
+		UserSiYuanOneTimePayStatus:     1,
+	}
+	return user, nil
+}
+
 func UseActivationcode(code string) (err error) {
 	code = util.RemoveInvalid(code)
 	requestResult := gulu.Ret.NewResult()
